src/main/java/uk/gov/hmcts/juror/job/execution/jobs/dashboard/ams/data/BureauLettersToBePrinted.java

```diff
@@ -12,37 +12,38 @@
-            '5224A','CONFIRMATION ENGLISH',
-            '5224AC','CONFIRMATION WELSH',
-            '5224C','WITHDRAWAL WELSH',
-            '5225','EXCUSAL ENGLISH',
-            '5225C','EXCUSAL WELSH',
-            '5226','NON-EXCUSAL ENGLISH',
-            '5226A','NON-DEFER ENGLISH',
-            '5226AC','NON-DEFER WELSH',
-            '5226C','NON-EXCUSAL WELSH',
-            '5227','REQUEST ENGLISH',
-            '5227C','REQUEST WELSH',
-            '5228','NON-RESPONDED ENGLISH',
-            '5228C','NON-RESPONDED WELSH',
-            '5229','POSTPONE ENGLISH',
-            '5229A','DEFERRED ENGLISH',
-            '5229AC','DEFERRED WELSH',
-            '5229C','POSTPONE WELSH',
-            'UNKNOWN') description,
-            sum(nvl(abc.number_of_items,0)) count
-            from form_attr f left outer join (
-            Select form_type, number_of_items  from abaccus a
-            where creation_date < trunc(sysdate) and
-            creation_date >= trunc(sysdate-decode(TO_CHAR(sysdate,'dy'), 'tue', 3,1))) abc
-            on f.form_type = abc.form_type
-            group by f.form_type, f.dir_name
-            order by 1
+    select tfa.form_type type,
+            (case when tfa.form_type ='5221' then 'SUMMONS ENGLISH'  
+            when tfa.form_type ='5221C' then 'SUMMONS BI-LINGUAL' 
+            when tfa.form_type ='5224' then 'WITHDRAWAL ENGLISH'
+            when tfa.form_type ='5224A' then 'CONFIRMATION ENGLISH'   
+            when tfa.form_type ='5224AC' then 'CONFIRMATION WELSH'   
+            when tfa.form_type ='5224C' then 'WITHDRAWAL WELSH'   
+            when tfa.form_type ='5225' then 'EXCUSAL ENGLISH'   
+            when tfa.form_type ='5225C' then 'EXCUSAL WELSH'   
+            when tfa.form_type ='5226' then 'NON-EXCUSAL ENGLISH'   
+            when tfa.form_type ='5226A' then 'NON-DEFER ENGLISH'   
+            when tfa.form_type ='5226AC' then 'NON-DEFER WELSH'   
+            when tfa.form_type ='5226C' then 'NON-EXCUSAL WELSH'   
+            when tfa.form_type ='5227' then 'REQUEST ENGLISH'   
+            when tfa.form_type ='5227C' then 'REQUEST WELSH'   
+            when tfa.form_type ='5228' then 'NON-RESPONDED ENGLISH'   
+            when tfa.form_type ='5228C' then 'NON-RESPONDED WELSH'   
+            when tfa.form_type ='5229' then 'POSTPONE ENGLISH'   
+            when tfa.form_type ='5229A' then 'DEFERRED ENGLISH'   
+            when tfa.form_type ='5229AC' then 'DEFERRED WELSH'   
+            when tfa.form_type ='5229C' then 'POSTPONE WELSH' else 'UNKNOWN' end) description,
+            sum(coalesce(abc.number_of_items,0)) "count"
+            from juror_mod.t_form_attr tfa left outer join (
+                SELECT form_type,
+                count(juror_no) AS number_of_items
+                FROM juror_mod.bulk_print_data bpd
+                where creation_date >= current_date - case when to_char(current_date, 'dy') = 'tue' then 3 else 1 end
```

Comment:
Why do we minus 3 days for tue but none for mon?

Reply:
I believe it's to do with letters not being printed over the weekend so it would include letters that were added to the queue on friday.